Update documentation for release v1.0 (#51)
CABLE-LSM/benchcab: docs/user_guide/index.md

```bash
benchcab -h
```

## Directory structure and files

The following files and directories are created when `benchcab -f` executes successfully:
```
.
├── benchmark_cable_qsub.sh
├── benchmark_cable_qsub.sh.o<jobid>
├── rev_number-1.log
├── runs
│   └── site
│       ├── logs
│       │   ├── <task>_log.txt
│       │   └── ...
│       ├── outputs
│       │   ├── <task>_out.nc
│       │   └── ...
│       └── tasks
│           ├── <task>
│           │   ├── cable (executable)
│           │   ├── cable.nml
│           │   ├── cable_soilparm.nml
│           │   └── pft_params.nml
│           └── ...
└── src
    ├── CABLE-AUX
    ├── <realisation-0>
    └── <realisation-1>
```

The `benchmark_cable_qsub.sh` file is the job script submitted to run the test suite and `benchmark_cable_qsub.sh.o<jobid>` contains the job's standard output/error stream.

The `rev_number-*.log` file keeps a record of the revision numbers used for each realisation specified in the config file.

The `src` directory contains the source code checked out from SVN for each branch specified in the config file (labelled `realisation-*` above) and the CABLE-AUX branch.

> The `runs/site` directory contains the log files, output files, and tasks for running CABLE. CABLE runs are organised into tasks where a task consists of a branch (realisation), a met file, and a science configuration. In the above directory structure, `<task>` uses the following naming convention:

Reviewer: ```suggestion
The `runs/site` directory contains the log files, output files, and tasks for running CABLE. CABLE runs are organised into tasks where a task consists of a branch (realisation), a meteorological forcing, and a science configuration. In the above directory structure, `<task>` uses the following naming convention:
```

```
<met_file_basename>_R<realisation_key>_S<science_config_key>
```

> where `met_file_base_name` is the base file name of the met file in the FLUXNET dataset, `realisation_key` is the branch key specified in the config file, and `science_config_key` identifies the science configuration used.

Reviewer: ```suggestion
where `met_file_base_name` is the base file name of the meteorological forcing file in the FLUXNET dataset, `realisation_key` is the branch key specified in the config file, and `science_config_key` identifies the science configuration used.
```

module load conda/analysis3-unstable
```

> Currently, `benchcab` can only run CABLE for single sites. To run the single site tests, run

Reviewer: ```suggestion
Currently, `benchcab` can only run CABLE for flux sites. To run the flux site tests, run
```
Otherwise, it's confusing with "forty-two site" and "five site" tests.

```bash
benchcab -f
```

The benchmarking will follow the steps:

1. Checkout both branches. The codes will be stored under `src/` directory in your work directory. The sub-directories are created automatically.
2. Compile the source code from both branches
3. Setup and launch a PBS job to run the simulations in parallel. When `benchcab` launches the PBS job, it will print out the job ID to the terminal. You can check the status of the job with `qstat`.

> The following directory structure is generated when simulations are run:

Author: See [d7221e7](https://github.com/CABLE-LSM/benchcab/pull/51/commits/d7221e71206d5a4f872760a81bc721549adc21f1). I've added a new section called "Directory structure and files" that which goes into more depth on all the other files/directories that are created.
